internal/datastore/mysql/migrations/zz_migration.0007_watch_api_index.go

```diff
@@ -0,0 +1,17 @@
+package migrations
+
+import "fmt"
+
+func addWatchAPIIndex(t *tables) string {
+	return fmt.Sprintf(`CREATE INDEX ix_watch
```

Comment:
`ix_caveat_watch`?

Reply:
oh boy, good you found that - that's the wrong table! 🙈 
See https://github.com/authzed/spicedb/issues/1543#issuecomment-1744739489